test/index.js

```diff
@@ -581,3 +601,10 @@ if (!module.parent) {
   exports.bench = bench;
   module.exports = exports;
 }
+
+// returns time to millisecond granularity
+function prettyElapsedTime(hrtimeElapsed) {
+  var seconds = hrtimeElapsed[0];
+  var fracInMs = Math.round(hrtimeElapsed[1] / 1e6);
```

Comment:
Not sure how hard up we are for the `ms` instead of seconds. Can we go to the second or third floating point??

```
0.001
```